tests/ts/liblastlog2/dlopen

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+TS_TOPDIR="${0%/*}/../.."
+TS_DESC="dlopen"
+
+. "$TS_TOPDIR"/functions.sh
+ts_init "$*"
+
+ts_check_test_command $TS_HELPER_LIBLASTLOG2_DLOPEN
+
+$TS_HELPER_LIBLASTLOG2_DLOPEN "$TS_TOPDIR"/../.libs/liblastlog2.so >/dev/null || ts_failed "returned an error"
```

Comment:
The test is failing during `distcheck`.
Unfortunately the output is discarded, so it's hard to know why.

It's weird that this test prints its failures to `stdout` in contrast to all the other tests which use `stderr`.